erc20-load-test-tool/script/run-dapp.py

```diff
@@ -0,0 +1,238 @@
+"""
```

Comment:
Follow PEP8 code style. Using editors like IntelliJ or PyCharm would suggest you to format code, remove parenthesis, etc.